server/routes/bulkOffchain.ts

```diff
@@ -122,8 +132,7 @@ const bulkOffchain = async (
           );
           const activeUsers = {};
           const where = { updated_at: { [Op.gt]: thirtyDaysAgo } };
-          if (community) where['community'] = community.id;
-          else where['chain'] = chain.id;
+          if (chain)where['chain'] = chain.id;
```

Comment:
Is it safe to now assume that chain will never be null here and else where using similar patterns?

Reply:
Yup, modifying the others as well